cocotb/scoreboard.py

```diff
@@ -99,6 +99,8 @@ def compare(self, got, exp, log, strict_type=True):
                       "transaction")
             log.info("Got: %s but expected %s" %
                      (str(type(got)), str(type(exp))))
+            log.info("Set strict_type=False when calling add_interface or "
```

Comment:
Because of parallelism, it might happen that these two lines are becoming separated in the output and are thus harder to interpret, so maybe better extend the first ``log.info()`` with this string.

Reply:
See f77661c for update.